requirements.txt

```diff
@@ -0,0 +1,10 @@
+# Python dependencies for building and testing the pytype source code.
+# Make sure you also install the non-Python dependencies described in
+# https://github.com/google/pytype/blob/master/CONTRIBUTING.md#pytype-dependencies.
+# In particular, use the official ninja package, not the pip distribution.
```

Comment:
if the pip installed ninja doesn't work, we should repeat this advice in CONTRIBUTING.md too.

Reply:
CONTRIBUTING.md links directly to ninja-build.org, which provides instructions on how to get the official package.

The only reason I mention the pip distribution here is that if someone checks setup.cfg, they might be confused by we have developers and users install ninja from different sources. Pip installing ninja won't cause any terrible problems, just make debugging harder in the unlikely event of the PyPI package (which isn't maintained by ninja, but by some ninja user that wanted ninja to be pip-installable for their own project) being broken or out-of-date.